### ISSUE TITLE
[FEATURE] - Codespaces Web: Have non-Codespaces projects open in local VS Code or VS Code insiders

### ISSUE THREAD

**Issue by TylerLeonhardt** [alefragnani/vscode-project-manager#485]
I think the title says it all. Right now, if you open a non-Codespaces project in Codespaces Web, it brings to you to the GitHub 404 page.

Instead, it would be cool if Codespaces Web could launch vscode/vscode insiders so that the experience is seemless.

**Comment by alefragnani** [comment]
Yep, it's that scenario we discussed. It will fit perfectly with #431 .

Also, I just noticed that if you try to open a Codespaces project in VS Code, but don't have the Codespaces extension installed/activated, it opens an "almost empty" window. I should give a better feedback on this one too.

Thanks for reporting

**Comment by alefragnani** [comment]
Hi @TylerLeonhardt ,

Yeah... it's been a while, I'm sorry, but since I'm planning to include #737 (and maybe #525) in the next release, I would like to also include this as well. I dig a little about it, but didn't find how to make Codespaces (or vscode.dev) to launch the VS Code desktop. I'm not sure it is just a matter of adding the proper scheme to the project's path, or something related to the `Continue On` feature.

Do you have any samples to provide direction? Know any extensions that already does that, so I could take a look?

Thank you
